feedapi/client.go

```diff
@@ -156,6 +157,9 @@ func (c Client) FetchEvents(ctx context.Context, token string, partitionID int,
 	if options.PageSizeHint != DefaultPageSize {
 		q.Add("pagesizehint", fmt.Sprintf("%d", options.PageSizeHint))
 	}
+	if options.EventTypes != nil {
+		q.Add("event-types", strings.Join(options.EventTypes, ";"))
+	}
```

Comment:
I wonder if, in the core feedapi-library, it would have been more appropriate to allow passing through a generic key-value parameters.

Perhaps the backend wants to add "color=blue" or "shoesize=42" as arguments too.

There is nothing the client/server library can do about the data, it is just about passing things through, so may as well just do it generic?

Reply:
The important bit for me is that it's supported.

I do prefer this myself because;

It's a bit more complex to make a generic mapping from a struct property 'EventTypes' to 'event-types' with a ';' delimiter than it is to do it specifically for the type.  

It's in line with what we do with the other parameters that's part of the spec already.

Reply:
So I would suggest team-postprocessing make it a backlog item to convert all 'pass-through' arguments to one generic solution if you feel that's a better option. I suppose the same applies for page-hint-size ect. :)

Reply:
I suppose it's also nice to have the actual property names available on the options struct, no?